src/models/user.ts

```diff
@@ -29,10 +29,10 @@ export class User {
   @Column({ type: 'varchar' })
   userType: UserType;
 
-  @Column({ nullable: true })
+  @Column({ nullable: true, unique: false })
```

Comment:
Without this you couldn't have a user with the same programId and institutionId

Reply:
its weird that its unique by default